api/v1_notifications.go

```diff
@@ -277,10 +277,25 @@ limit @limit::int
 				action, _ = sjson.SetBytes(action, "data.is_album", isAlbum)
 			}
 
-			// amount + tip_amount: to_wei_string
-			for _, fieldPath := range []string{"data.amount", "data.tip_amount"} {
-				if val := gjson.GetBytes(action, fieldPath); val.Exists() {
-					action, _ = sjson.SetBytes(action, fieldPath, val.String()+"0000000000")
+			// For notifications in $AUDIO, we need to add 0000000000 to the amount field
+			// to convert from SPL to wei
+			if notif.Type == "tip_send" ||
+				notif.Type == "tip_receive" ||
+				notif.Type == "challenge_reward" ||
+				notif.Type == "claimable_reward" ||
+				notif.Type == "reaction" {
+				for _, fieldPath := range []string{"data.amount", "data.tip_amount"} {
```

Comment:
oh man, this is totally going to bite us when/if we do tipping/rewards with artist coins huh.

Reply:
yes... i'll think of a better long term plan

Reply:
maybe these notifs should not be in WEI at all